packages/website/components/Topbar.tsx

```diff
@@ -92,7 +92,7 @@ export const Topbar = () => (
           <TopbarLink href="/tokens/color-system" label="Tokens" />
         </li>
         <li>
-          <TopbarLink href="/components/box" label="Components" />
+          <TopbarLink href="/components/accordion" label="Components" />
```

Comment:
Hmmm, I'm not a biggest fan of `/accordion` being the first page 🤔

Reply:
It's the first one when we order items alphabetically 🙂